fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -230,15 +232,15 @@ class FMCMainDisplay extends BaseAirliners {
             this.arincLandingElevation,
             this.arincDestinationLatitude,
             this.arincDestinationLongitude,
+            this.arincMDA,
+            this.arincDH,
             this.arincThrustReductionAltitude,
             this.arincAccelerationAltitude,
-            this.arincEoAccelerationAltitude,
```

Comment:
Oh, wait. Don't remove this one!

Reply:
wait why is gone, jeez.

Reply:
at least the plane does not crash without it, when u do a failed merge conflict.

Reply:
Yeah, just conflict resolution oopsie. Add it back and this PR is good to go from me. 👍